### PR TITLE
Update ui again

### PR DESCRIPTION
**Pull request by maryx**
- Updates UI for homepage and dropdowns

### CODE REVIEW

File: unit_converter/unit_converter/lib/converter_route.dart [hunk 1]
```diff
@@ -10,14 +10,10 @@ import 'package:unit_converter/api.dart';
 import 'package:unit_converter/category_route.dart';
 import 'package:unit_converter/unit.dart';
 
-const _padding = EdgeInsets.all(16.0);
+final _padding = EdgeInsets.all(16.0);
```
Comment (yjbanov) [inline unit_converter/unit_converter/lib/converter_route.dart]: Curious why did you have to change this to `final`?

**Comment by yjbanov** [comment]
lgtm

File: unit_converter/unit_converter/lib/category.dart [hunk 2]
```diff
@@ -9,9 +9,10 @@ import 'package:flutter/material.dart';
 import 'package:unit_converter/converter_route.dart';
 import 'package:unit_converter/unit.dart';
 
-// We use an underscore to indicate that the border radius is private.
+// We use an underscore to indicate that these variables are private.
 // See https://www.dartlang.org/guides/language/effective-dart/design#libraries
-final _borderRadius = BorderRadius.circular(4.0);
+final _rowHeight = 100.0;
+final _borderRadius = BorderRadius.circular(_rowHeight / 2);
```
[the remark this answers is not in the document]
Comment (maryx) [reply unit_converter/unit_converter/lib/category.dart]: Right, thanks! The BorderRadius says it is not a const constructor (even if I pass in a hardcoded double).

File: unit_converter/unit_converter/lib/converter_route.dart (hunk 1 above)
Comment (maryx) [reply unit_converter/unit_converter/lib/converter_route.dart]: Done.

**Pull request merged**